test/test_transforms.py

```diff
@@ -462,6 +462,15 @@ def test_resize_size_equals_small_edge_size(height, width):
     result = t(img)
     assert max(result.size) == max_size
 
+def test_resize_equal_input_output_sizes():
```

Comment:
There is a blank line missing here: https://github.com/pytorch/vision/actions/runs/4762922184/jobs/8465723356?pr=7519

```suggestion

def test_resize_equal_input_output_sizes():
```